subject/biology-course.html

```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+    

```

Comment:
same?...why do we have so many <br>?

Reply:
idk....

Reply:
let's keep them later on we can delete them